### PR TITLE
Update docs navigation

### PR DESCRIPTION
**Pull request by kdabo**
- Hidden the level 3 sub sections and moved them as tabs of the level 2 sections.
- Updated the navigation structure to the following:
```
- Behaviours
	- Hovers
	- Cursor
- Components
	- Button 
	- Button Group
	- Checkbox
	- Dropdown
	- Form Control
	- Header
	- Icons
	- Input
	- Label
	- Loader
	- Preview
	- Select
	- Table List
	- Tile
	- Toast
	- Upload
- Layout
	- Containers
	- Grid
	- Flex
	- Responsive
	- Spacing
- Style
	- Colors
	- Typography
```
## Behaviours
screen shot 2017-11-15 at 11 56 47

## Components
screen shot 2017-11-15 at 11 56 59

## Layout
screen shot 2017-11-15 at 11 57 12

## Styles
screen shot 2017-11-15 at 11 57 27

### CODE REVIEW

File: docs-builder/index.hbs [hunk 1]
```diff
@@ -21,33 +21,28 @@
...
+                    <span class="kss-nav__name">{{header}}</span>
                 </a>
-              </li>
-            {{/each}}
-            </ul>
-          {{/if}}
-        {{/if}}
-      </li>
-    {{/each}}
+                {{#if isActive}}
```
Comment (kagnamazian) [inline docs-builder/index.hbs]: Can't you do  `{{#if isActive && children}}` ?

File: docs-builder/kss-assets/kss.scss [hunk 2]
```diff
@@ -266,17 +274,28 @@ $kss-vertical-rhythm      : $kss-font-size * 1.5;
       display: list-item;
       padding-right: 0;
     }
+
+    .kss-section--depth-2,
+    .kss-section--depth-3 {
+      text-transform: capitalize !important;
```
Comment (kagnamazian) [inline docs-builder/kss-assets/kss.scss]: oula ! no choice that we had  to put `!important` ?

Comment (enbajo) [reply docs-builder/kss-assets/kss.scss]: You're right, and also the `kss-section--depth-3` is not needed, they're leftovers from the previous commit.

File: docs-builder/index.hbs (hunk 1 above)
Comment (enbajo) [reply docs-builder/index.hbs]: Unfortunately there is not way in handlebars to incorporate logical operators into the standard conditional operator 😕

**Comment by enbajo** [comment]
Yeah I opted for not hacking it as the page was also flashing every time you changed the section. I talked with Gal about keeping it in a single page as it is currently and she was fine with it. I'll put some space and a line at the end of each section.

Unfortunately the kss generates everything in the same page and the only way to change the layout is by hacking it. I think moving forward we should go for more flexible options such as:

- https://github.com/straker/livingcss
- https://github.com/DSSWG/DSS

These will return a JSON representation of the CSS docs which gives us more flexibility on how we want to display the styleguide (we can potentially use React).

> Can we also have the top level headings (behaviour, components etc) have a capital first letter?
You mean the ones in the sidebar menu? The headings in the page already have a capital first letter

**Comment by beerecca** [comment]
👍 

No on the page - this is what I get:
screen shot 2017-11-17 at 12 40 26

**Comment by enbajo** [comment]
@beerecca 

The 3rd level menu shouldn't exist, have you pulled the latest?

**Comment by beerecca** [comment]
Nice!

**Pull request merged**